docs/docs/how-to/using-gitpod.md

```diff
@@ -0,0 +1,58 @@
+# Using Gitpod
+
+## What is Gitpod?
+Gitpod is a cloud development environment with all the necessary tools and dependencies, allowing you to focus on building your RedwoodJS application without worrying about the setup. Get started quickly and efficiently by launching RedwoodJS inside Gitpod!
+
+## Getting Started
+Click on the Open in Gitpod button:
+
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/redwoodjs/starter)
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/guz67aa_1Wk?si=p1uc2EK6o8HJGBax" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+This will launch GitPod and ask you to configure a new workspace. Click continue.
+
+![GitPod Onboarding Screen](https://github.com/redwoodjs/starter/raw/main/images/gitpod-new-workspace.png)
+
+GitPod will then begin to build your workspace. This may take several minutes.
+
+What's going on behind the scenes:
+
+- GitPod is setting the workspace set up
+- It installs our recommended VS Code plugins:
+  - [ESLint](https://github.com/redwoodjs/starter/blob/main)
+  - [Git Lens](https://github.com/redwoodjs/starter/blob/main)
+  - [VS Code Language - Babel](https://github.com/redwoodjs/starter/blob/main)
+  - [VS Code Version Lens](https://github.com/redwoodjs/starter#:~:text=VS%20Code%20Version%20Lens)
+  - [Editor Config](https://github.com/redwoodjs/starter#:~:text=Code%20Version%20Lens-,Editor%20Config,-Prisma)
+  - [Prisma](https://github.com/redwoodjs/starter/blob/main)
+  - [VS Code GraphQL](https://github.com/redwoodjs/starter/blob/main)
+- It runs our **Create Redwood App** which will install the latest stable version of Redwood. We're setting this project to use TypeScript, however, you can [change it to JavaScript](https://github.com/redwoodjs/starter/blob/main) if you prefer.
+- It runs `yarn install`, adding all the dependencies for the project
+Changes the database to a postgres database
+
+Once everything is up and running, you can click on the Ports tab
+
+[![GitPod Ports Tab](https://github.com/redwoodjs/starter/blob/main/images/gitpod-ports.png)]
```

Comment:
This image isn't rendering in the docs for some reason:

<img width="631" alt="image" src="https://github.com/redwoodjs/redwood/assets/32992335/2f303a0a-5cfd-43f0-99f8-98d7953234ef">